modules/indexing/src/main/java/org/apache/ignite/internal/mxbean/SqlQueryMXBeanImpl.java

```diff
@@ -73,19 +74,30 @@ public SqlQueryMXBeanImpl(GridKernalContext ctx) {
         h2idx.longRunningQueries().setResultSetSizeThresholdMultiplier(rsSizeThresholdMultiplier);
     }
 
+    /** {@inheritDoc} */
+    @Override public Long getSqlGlobalMemoryQuotaBytes() {
+        return U.parseBytes(h2idx.memoryManager().getGlobalQuota());
```

Comment:
We already have an integer value of the quota, right? So we probably don't need `parseBytes`.

May change the `memoryManager().getGlobalQuota()` to return `long` instead of String? It is internal and doesn't seem to be used anywhere else, so we can change it easily.